### ISSUE TITLE
Settings: Label of “Play animated...” is cut off

### ISSUE THREAD

**Issue by realpixelcode** [mastodon/mastodon-android#55]
In the settings, the label of the option to play animated avatars and emoji is cut off so one can't read it in full. While it's possible to read the whole label by switching to landscape mode, the label's text should just break into a new line.

![Screenshot-2022-04-20-00-42-57.png](https://user-images.githubusercontent.com/52963327/164113681-47a10bf8-311d-48e2-a4f4-af1cc2c863e7.png)

**Comment by grishka** [comment]
> the label's text should just break into a new line.

No it shouldn't. This would look terrible.

**Comment by realpixelcode** [comment]
> > the label's text should just break into a new line.
> 
> No it shouldn't. This would look terrible.

But a cut-off label does not, I guess? 🤔

**Comment by grishka** [comment]
As I like to tell designers: the screen size of an Android device is two random numbers.

![Screenshot_20220420-153040.png](https://user-images.githubusercontent.com/1478258/164230820-241ae332-6af2-4b73-b12f-bcb33b9d0452.png)

**Comment by Brawaru** [comment]
Consider: users that use larger font sizes because they have low vision. I know one person who does that — my mother, there are certain apps that she literally cannot use because they do not care about impaired users.

Also consider: other languages. As someone speaking russian, you should be aware how lengthy cyrillic languages are at times.

![Screenshot of toggles ‘Play animated avata…’ and ‘Use in-app browser’ with large font size](https://user-images.githubusercontent.com/10401817/164441839-7d37111a-4745-42cb-b9fd-823952136524.png)

Here's Discord as example, feeling pretty okay wrapping the toggles:
![Screenshot of toggles in Discord, the text is too lengthy to fit in a single line and wrapped to a next line](https://user-images.githubusercontent.com/10401817/164443381-918304b1-93c7-48d7-b34c-bc2341dcab2f.png)

**Comment by sk22** [comment]
> No it shouldn't.

Yes it should, see [@Brawaru's comment](https://github.com/mastodon/mastodon-android/issues/55#issuecomment-1105057553)

> This would look terrible.

It does, but it's also necessary.

For reference, it can look something like this:
<img width=300 src="https://user-images.githubusercontent.com/6217438/217488487-f1e8e584-fec6-49d0-8f9d-80e93bf0a9f3.png">

Except for text to scale properly, the app would have to use "sp" instead of "dp" for text sizes, which would be another big accessibility concern.

**Comment by realpixelcode** [comment]
It's been fixed.

![Screenshot-2026-05-16-16-53-57.jpeg](https://github.com/user-attachments/assets/27ed0c09-52be-4e62-8b6c-be8227f441b5)
